jekyll/_config_staging.yml

```diff
@@ -1 +1 @@
-url: "https://static-staging.circleci.com"
+url: "https://cci.feliciano.tech"
```

Comment:
Staging is removed, at this point, this is just a string. Does nothing. Do you want me to keep the previous string instead? Neither does anything but I can do a final commit here if you want.